astro/src/content/docs/lifecycle/authenticate-users/identity-providers/social/linkedin.mdx

```diff
@@ -61,13 +62,24 @@ Add your redirect URL. This value will be your FusionAuth installation hostname
 
 Save the setting.
 
-You should see the following scopes in the <strong>Oauth 2.0 Scopes</strong> section:
+You should see the following scopes in the <strong>OAuth 2.0 Scopes</strong> section:
 
-* `r_emailaddress`
-* `r_liteprofile`
+* `openid`
+* `profile`
+* `email`
 
-<Aside type="note">
-If you do not see these scopes, you may not have added the LinkedIn product to your LinkedIn app correctly. You may also need to reload the app page or log out of LinkedIn and log in again to make sure your application has the proper permissions.
+If these are not there you may need to add them to FusionAuth and trying a login first. See [Create a LinkedIn Identity Provider](#create-a-linkedin-identity-provider).
```

Comment:
I'm not sure what "add them to FusionAuth and try a login first" means. Could you make this clearer, and maybe explain why they might not be there?